app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -93,6 +93,12 @@ export class FollowUsers extends Component {
     }
   }
 
+  handleKeyPress(event, user) {
+    if (event.key === ' ') {
+      this.handleClick(user);
+    }
+  }
+
```

Comment:
refactor (blocking) - I don't think this is used anywhere/needed so let's delete